projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/ui-tests/tests/vdk-jupyterlab-extension.spec.ts

```diff
@@ -1,21 +1,125 @@
 import { expect, test } from '@jupyterlab/galata';
+import { assert } from 'console';
 
 /**
  * Don't load JupyterLab webpage before running the tests.
  * This is required to ensure we capture all log messages.
  */
 test.use({ autoGoto: false });
+const delay = ms => new Promise(res => setTimeout(res, ms));
 
-test('should emit an activation console message', async ({ page }) => {
-  const logs: string[] = [];
+test('should open run job pop up and then cancel the operation', async ({ page }) => {
+  await page.goto("");
+  await page.menu.open('VDK');
+  await page.locator('#jp-vdk-menu').getByText('Run').click();
+  await page.locator('div').filter({ hasText: 'Run Job' });
+  await page.getByRole('button', { name: 'Cancel' }).click();
+});
+
+
+test('should try to run a job with empty input and get error', async ({ page }) => {
+  await page.goto("");
+  await page.menu.open('VDK');
+  await page.locator('#jp-vdk-menu').getByText('Run').click();
+  await page.locator('div').filter({ hasText: 'Run Job' });
+  await page.getByRole('button', { name: 'OK' }).click();
+  await page.locator('div').filter({ hasText: 'Encountered an error when trying to run the job.' });
+  await page.getByRole('button', { name: 'OK' }).click();
+
+});
 
-  page.on('console', message => {
-    logs.push(message.text());
+test('should try to run a job with incorrect data and get a dialog error message', async ({page}) =>{
```

Comment:
This is completely unrelated, but shouldn't the bot be configured to re-format these? There are other places in the code where with no space before the opening brace of a function and the bot just lets it slide. 🤔

Reply:
I fixed it manually to push merge this PR, but the topic should be discussed further.